src/components/ProductTours/AdminOnboardingTours/OnboardingWelcomeModal.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  ActionRow, Button, Image, ModalDialog, useToggle,
+} from '@openedx/paragon';
+import { FormattedMessage, useIntl } from '@edx/frontend-platform/i18n';
+
+import { ONBOARDING_WELCOME_MODAL_COOKIE_NAME } from './constants';
+import messages from './messages';
+import welcomeModal from '../data/images/WelcomeModal.svg';
+
+const OnboardingWelcomeModal = ({
+  openAdminTour, lastLogin,
+}) => {
+  const intl = useIntl();
+  const [isOpen, open, close] = useToggle(false);
+  const isNewUser = lastLogin == null;
```

Comment:
nit: This is what I should have suggested earlier!
```suggestion
  const isNewUser = !lastLogin;
```